web/css/index.css

```diff
@@ -530,7 +530,7 @@ header.active {
 }
 
 #moz-links .links a {
-  color: var(--color-help);
+  color: #b7d43f;
```

Comment:
We have a css variable for this called `--color-listen`, let's use that here instead of hard-coding the value here.